pkg/install/go.mod

```diff
@@ -8,4 +8,6 @@ require (
 	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/tools v0.0.0-20200121210457-b3205ff6fffe // indirect
+	k8s.io/api v0.17.2
```

Comment:
Any idea how these dependencies end up here? On the surface of it, it doesn't seem like changes in the module code would prompt the change here.

Reply:
Darn, that was probably because of #2652 

The only reason for this was unmarshalling a deployment. I think I could use an anonymous struct with the relevant fields. But I would rather do it in a separate PR.

Reply:
It's getting hard to keep `pkg/install` kubernetes-free

Reply:
OK, in what order do you want to remedy these things

Reply:
Let's merge this first

I fixed it at https://github.com/fluxcd/flux/pull/2824

Reply:
Or rather, I don't care about the ordering. I will deal with conflicts if I have to.

Reply:
I rebased on master. So this is fully resolved now.